cfpb/consumerfinance.gov
# Cards: Fix HTML validator issue with div inside of h2

## Changes

- Change card's div inside an h2 to a span.


## How to test this PR

1. Pull branch and run site.
2. Inspect page and go to HTML Validator tab and click "W3c online" 
3. Compare results to live site and see that errors about a div inside an h2 are gone.
4. The cards on the bottom of the homepage on this branch and live site should look and function the same.
5. The cards on https://www.consumerfinance.gov/ask-cfpb/ and this branch look and function the same.


## Screenshots

Before:

<img width="743" alt="Screen Shot 2020-08-21 at 5 50 20 PM" src="https://user-images.githubusercontent.com/704760/90938313-d12b3e00-e3d6-11ea-9308-40c53351645c.png">
